dart-impl/mpi/src/dart_team_group.c

```diff
@@ -27,9 +27,10 @@
 
 static struct dart_group_struct* allocate_group()
 {
-  struct dart_group_struct* group = malloc(sizeof(struct dart_group_struct));
+  struct dart_group_struct* group = (struct dart_group_struct *) malloc(
```

Comment:
@devreal : of course you are right ;) The problems are related to a mismatch of `dart_unit_t` and `dart_global_unit_t` and the cast hides them. `clang-analyze` perfectly accepts malloc without cast iff the types are equal.